checker/tests/index/RepeatLTLengthOf.java

```diff
@@ -0,0 +1,73 @@
+import org.checkerframework.checker.index.qual.EnsuresLTLengthOf;
+import org.checkerframework.checker.index.qual.EnsuresLTLengthOfIf;
+
+class RepeatLTLengthOf {
+
+    protected String value1;
+    protected String value2;
+    protected String value3;
+    protected int v1;
+    protected int v2;
+    protected int v3;
+
+    public void func1() {
+        v1 = value1.length() - 1;
+        v2 = value2.length() - 3;
+        v3 = value3.length() - 4;
+    }
+
+    public boolean func2() {
+        v1 = value1.length() - 3;
```

Comment:
Why are these out of order?  That adds extra cognitive load to anyone reading the code.

Reply:
@mernst What do you mean by out of order?

Reply:
The integer constants (the subtrahends) are out of order.

Reply:
Fixed!